jekyll/_includes/segment.html

```diff
@@ -1,5 +1,8 @@
 <script type="text/javascript">
-  !function(){var analytics=window.analytics=window.analytics||[];if(!analytics.initialize)if(analytics.invoked)window.console&&console.error&&console.error("Segment snippet included twice.");else{analytics.invoked=!0;analytics.methods=["trackSubmit","trackClick","trackLink","trackForm","pageview","identify","reset","group","track","ready","alias","debug","page","once","off","on"];analytics.factory=function(t){return function(){var e=Array.prototype.slice.call(arguments);e.unshift(t);analytics.push(e);return analytics}};for(var t=0;t<analytics.methods.length;t++){var e=analytics.methods[t];analytics[e]=analytics.factory(e)}analytics.load=function(t){var e=document.createElement("script");e.type="text/javascript";e.async=!0;e.src=("https:"===document.location.protocol?"https://":"http://")+"cdn.segment.com/analytics.js/v1/"+t+"/analytics.min.js";var n=document.getElementsByTagName("script")[0];n.parentNode.insertBefore(e,n)};analytics.SNIPPET_VERSION="4.0.0";
+  !function(){
+    var isProduction = window.location.origin === 'https://circleci.com';
```

Comment:
Do browsers automatically remove www. ?

Reply:
do we want to inline this with the rest of the script?

Reply:
we do not have `www` on circleci.com:

![Screen Shot 2021-12-06 at 12 48 02 PM](https://user-images.githubusercontent.com/1449325/144920189-fd3b1c3d-2e48-4c9c-a370-4b11a7509c3e.png)

Reply:
@nramabad I don't think so. line `5` was the original snippet and line 3-4 is the additional "patch" we introduced. I'll add a comment tho